test/functional/Valhalla/src_qtypes/org/openj9/test/lworld/ValueTypeSystemArraycopyTests.java

```diff
@@ -313,38 +285,12 @@ static private void testOBJVT(SomeValueClass[] dst) {
 		checkResults(src, dst);
 	}
 
-	static private void testPVTOBJ(SomePrimitiveValueClass[] src) {
```

Comment:
After this is removed, do we have test cases of `System.arraycopy()` between `Object[]` and null-restricted array ?

Reply:
I thought testSystemArrayCopy15 would still do that since a null-restricted array will be passed in to testVTOBJ.